src/f5.go

```diff
@@ -54,6 +54,7 @@ func main() {
 
 	err = client.LogIn()
 	exitOnErr(err)
+	defer exitOnErr(client.LogOut())
```

Comment:
not sure about the os.Exit preventing this to be executed if publish fails, but it maybe safer if we do the logout after the collectEntities

Reply:
fixed